bundle/src/lib/header-bidding/prebid-types.ts

```diff
@@ -79,10 +79,11 @@ type PrebidOpenXParams = {
 
 type PrebidCriteoParams =
 	| {
-			networkId: number;
+			zoneId?: number;
```

Comment:
Not sure this should be optional, an object of type `PrebidCriteoParams` could be either an object with `zoneId` or an object with `networkId` and an optional `pubId`. This type with `zoneId?` implies it could be an empty object? Might be worth testing to verify that.